### PR TITLE
Change some error messages in mbt build / module build

### PR DESCRIPTION
**Pull request by tal-sapan**
## Description

Jira: DEVXBUGS-4924

### Checklist
- [X] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [X] Relevant logs were added
- [X] Formatting and linting run locally successfully
- [X] All tests pass
- [x] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

**Comment by tal-sapan** [comment]
Note: not reviewed by UA yet

File: internal/exec/exec.go [hunk 1]
```diff
@@ -91,12 +95,12 @@ func executeCommand(cmd *exec.Cmd, terminateCh <-chan struct{}) error {
 	// During the running process get the standard output
 	stdout, err := cmd.StdoutPipe()
 	if err != nil {
-		return errors.Wrapf(err, execFailedOnStdoutMsg, cmd.Path)
+		return errors.Wrapf(err, execFailedOnStdoutMsg)
```
Comment (allaVolkov) [inline internal/exec/exec.go]: Wrap is enough

File: internal/exec/exec.go [hunk 2]
```diff
@@ -91,12 +95,12 @@ func executeCommand(cmd *exec.Cmd, terminateCh <-chan struct{}) error {
...
 	}
 	// During the running process get the standard output
 	stderr, err := cmd.StderrPipe()
 	if err != nil {
-		return errors.Wrapf(err, execFailedOnStderrMsg, cmd.Path)
+		return errors.Wrapf(err, execFailedOnStderrMsg)
```
Comment (allaVolkov) [inline internal/exec/exec.go]: Wrap

File: internal/exec/exec.go [hunk 3]
```diff
@@ -118,13 +122,13 @@ func executeCommand(cmd *exec.Cmd, terminateCh <-chan struct{}) error {
 		// Note: this waits until the process finishes or an error occurs.
 		scanout, scanerr := scanner(stdout, stderr)
 
-		if scanerr.Err() != nil {
-			finishedCh <- errors.Wrapf(err, execFailedOnScanMsg, cmd.Path)
+		if err := scanerr.Err(); err != nil {
+			finishedCh <- errors.Wrapf(err, execFailedOnScanerrMsg)
```
Comment (allaVolkov) [inline internal/exec/exec.go]: Wrap

File: internal/exec/exec.go [hunk 4]
```diff
@@ -118,13 +122,13 @@ func executeCommand(cmd *exec.Cmd, terminateCh <-chan struct{}) error {
...
 			return
 		}
 
-		if scanout.Err() != nil {
-			finishedCh <- errors.Wrapf(err, execFailedOnErrorGetMsg, cmd.Path)
+		if err := scanout.Err(); err != nil {
+			finishedCh <- errors.Wrapf(err, execFailedOnScanoutMsg)
```
Comment (allaVolkov) [inline internal/exec/exec.go]: Wrap

File: internal/artifacts/project.go [hunk 5]
```diff
@@ -32,16 +32,19 @@ func ExecBuild(makefileTmp, buildProjectCmdSrc, buildProjectCmdTrg, buildProject
...
+	if removeError != nil {
+		removeError = errors.Wrapf(removeError, removeFailedMsg, makefileTmp)
 	}
-	return err
+
+	if err != nil {
+		if removeError != nil {
+			logs.Logger.Error(removeError)
+		}
+		return errors.Wrapf(err, execFailedMsg)
```
Comment (allaVolkov) [inline internal/artifacts/project.go]: Wrap

File: internal/exec/exec_msg.go [hunk 6]
```diff
@@ -1,17 +1,16 @@
...
-	execMsg                 = `executing the "%s" command...`
-	execFileMsg             = `the executable file is "%s"`
-	execFailedOnStdoutMsg   = `could not execute the "%s" command when getting the stdout pipe`
-	execFailedOnStderrMsg   = `could not execute the "%s" command when getting the stderr pipe`
-	execFailedOnStartMsg    = `could not execute the "%s" command when starting`
-	execFailedOnScanMsg     = `could not execute the "%s" command when scanning the stdout and stderr pipes`
-	execFailedOnErrorGetMsg = `could not execute the "%s" command when receiving an error from the scanout object`
-	execFailedOnFinishMsg   = `could not execute the "%s" command when waiting for finish`
+	execMsg                = `executing the "%s" command...`
+	execFileMsg            = `the executable file is "%s"`
```
Comment (Paolaufer) [inline internal/exec/exec_msg.go]:
```suggestion
	execFileMsg            = `the executable file is at "%s"`
```

File: internal/exec/exec_msg.go [hunk 7]
```diff
@@ -1,17 +1,16 @@
...
+	execFailed             = `could not execute the "%s" command`
+	execFailedOnStdoutMsg  = `could not get the stdout pipe`
```
Comment (Paolaufer) [inline internal/exec/exec_msg.go]:
```suggestion
	execFailedOnStdoutMsg  = `could not get the "stdout" pipe`
```

File: internal/exec/exec_msg.go [hunk 8]
```diff
@@ -1,17 +1,16 @@
...
+	execFailedOnStderrMsg  = `could not get the stderr pipe`
+	execFailedOnScanerrMsg = `could not read from the stderr pipe`
+	execFailedOnScanoutMsg = `could not read from the stdout pipe`
 	// ExecInvalidTimeoutMsg is the error message that occurs when a timeout value is invalid
 	ExecInvalidTimeoutMsg = `invalid timeout value "%s", it should be in the form "[123h][123m][123s]"`
 	// ExecTimeoutMsg is the error message that occurs when a timeout is reached during commands execution
-	ExecTimeoutMsg = `commands timed out after %s`
-	execKilledMsg  = `process was terminated`
+	ExecTimeoutMsg = `build timed out after %s`
+	execKilledMsg  = `process was interrupted`
```
Comment (Paolaufer) [inline internal/exec/exec_msg.go]:
```suggestion
	execKilledMsg  = `the process was interrupted`
```

File: internal/exec/exec_msg.go [hunk 9]
```diff
@@ -1,17 +1,16 @@
...
+	execFailedOnStdoutMsg  = `could not get the stdout pipe`
+	execFailedOnStderrMsg  = `could not get the stderr pipe`
+	execFailedOnScanerrMsg = `could not read from the stderr pipe`
+	execFailedOnScanoutMsg = `could not read from the stdout pipe`
 	// ExecInvalidTimeoutMsg is the error message that occurs when a timeout value is invalid
 	ExecInvalidTimeoutMsg = `invalid timeout value "%s", it should be in the form "[123h][123m][123s]"`
 	// ExecTimeoutMsg is the error message that occurs when a timeout is reached during commands execution
-	ExecTimeoutMsg = `commands timed out after %s`
-	execKilledMsg  = `process was terminated`
+	ExecTimeoutMsg = `build timed out after %s`
```
Comment (Paolaufer) [inline internal/exec/exec_msg.go]:
```suggestion
	ExecTimeoutMsg = `the build timed out after %s`
```

File: internal/artifacts/project.go (hunk 5 above)
Comment (tal-sapan) [reply internal/artifacts/project.go]: Done

File: internal/exec/exec.go (hunk 4 above)
Comment (tal-sapan) [reply internal/exec/exec.go]: Done

File: internal/exec/exec.go (hunk 3 above)
Comment (tal-sapan) [reply internal/exec/exec.go]: Done

File: internal/exec/exec.go (hunk 2 above)
Comment (tal-sapan) [reply internal/exec/exec.go]: Done

File: internal/exec/exec.go (hunk 1 above)
Comment (tal-sapan) [reply internal/exec/exec.go]: Done

**Pull request merged**